packages/vuetify/src/components/VOtpInput/VOtpInput.sass

```diff
@@ -15,6 +15,11 @@
     .v-field
       height: 100%
 
+      .v-field__outline
+        &__start,
+        &__end
+          flex: 1
```

Comment:
This was required because `v-field__outline__start` wasn't occupying enough horizontal space for the border radius to be visible. SInce in VOptInput label won't be shown with VField, this shouldn't make any difference.

Reply:
The reason why `v-field__outline__start` wasn't occupying enough horizontal space on VOtpInput was because of this code. A better solution could be to make necessary change here. Any insights are appreciated. @KaelWD 

![image](https://github.com/user-attachments/assets/5538891a-aa3b-489d-a99f-72ab5a9590ff)

Reply:
It feels clean enough to me